src/main/java/gaffer/cli/StartCommand.java

```diff
@@ -52,12 +57,14 @@ public StartCommand() {
   @Override
   public void execute() throws CommandException {
     final Path path = Paths.get(flagProcfile).toAbsolutePath();
+    final Path envPath = Paths.get(flagEnvironment).toAbsolutePath();
     try {
       final Map<String, Integer> concurrency = ConcurrencyFlagParser.parse(flagConcurrency);
       final Procfile pf = Procfile.read(path);
-      new ProcessManager().start(pf, process, concurrency, flagPort);
+      final Environment env = Environment.read(envPath);
```

Comment:
I think we ignore the `.env` file if it doesn't exist instead of throwing exception. That's how `foreman` or `forego` work.